src/luerl.erl

```diff
@@ -418,3 +425,13 @@ decode_table(#tref{i=N}=T, St, In0) ->
 decode_userdata(U, St) ->
     {#userdata{d=Data},_} = luerl_heap:get_userdata(U, St),
     {userdata,Data}.
+
+%% Externalize and Internalize ensure that the VM state passed in
+%% can be stored externally or can be recreated from external storage.
+%% Currently very simple: only random state needs special treatment.
```

Comment:
What does this mean?

Reply:
See https://luerl.slack.com/archives/C50BCHSUW/p1688420734194269?thread_ts=1688419477.258639&cid=C50BCHSUW (once this is merged, I'll propose some Wiki additions)